### PR TITLE
stripping extra characters that cause problems

### PR DESCRIPTION
**Pull request by dan9186**
stripping off http/https if it's included

### CODE REVIEW

File: modules/golang/golang [hunk 1]
```diff
@@ -16,6 +16,10 @@ go () {
 			cd `go list -f '{{.Dir}}' ".../$1" | head -n 1`
 		fi
 	;;
+	"get")
+		shift
+		command go get $(echo $1 | sed 's/^http.*:\/*//')
```
Comment (brimstone) [inline modules/golang/golang]: This should probably be `https?://` not `http.*://` I'm not sure if that needs `sed -E` or something though.

Comment (dan9186) [inline modules/golang/golang]: Maybe that was the option I was missing.  When I tried `https?` on a Mac it would not match.

**Comment by brimstone** [comment]
@dan9186 did `sed -E` fix it? This seems to work on debian well enough.

**Comment by dan9186** [comment]
Yes. Fixed for Mac quite nicely. 

Daniel

**Comment by brimstone** [comment]
:+1:

**Pull request merged**